packages/workbox-routing/src/_types.ts

```diff
@@ -6,7 +6,17 @@
   https://opensource.org/licenses/MIT.
 */
 
-import './_version.mjs';
+import './_version.js';
+
+export interface MatchCallbackOptions {
+  url: URL;
+  request?: Request;
```

Comment:
I *think* `request` is meant to be required. Is there a place in the codebase that we use `MatchCallbackOptions` where `request` isn't set?

Reply:
Not that I can see, but in our current docs [it's listed as being optional](https://developers.google.com/web/tools/workbox/reference-docs/latest/workbox.routing.Route#~matchCallback).

Reply:
I think it should be required. Let's switch and see if anyone ends up complaining...

Reply:
Looks like [`request` is required in `findMatchingRoute()`](https://developers.google.com/web/tools/workbox/reference-docs/latest/workbox.routing.Router#findMatchingRoute), so at least in terms of our code it seems to be always used.

I suppose there's no harm in changing it to say it's now always passed. If someone wasn't using it before it won't hurt.